_content/monographs/logic-epistemology-theravada_hegoda-khemananda.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Logic and Epistemology in Theravada"
+authors:
+  - "Hegoda Khemananda"
+external_url: "https://archive.org/download/logic-and-epistemology-in-theravada/Logic%20and%20Epistemology%20in%20Theravada_text.pdf"
```

Comment:
Please also include the `source_url: "http://www.kbrl.gov.mm/book/details/002683"`